shared/graphql/queries/user/getUserCommunityConnection.js

```diff
@@ -47,7 +47,7 @@ const getUserCommunityConnectionOptions = {
 export const getCurrentUserCommunityConnection = graphql(
   getCurrentUserCommunityConnectionQuery,
   {
-    options: { fetchPolicy: 'cache-and-network' },
+    options: { fetchPolicy: 'cache-first' },
```

Comment:
Reviewers note: now that we have a subscription for this data there is no need to fetch this on every page load. New stuff will come in anyway.